requests_hawk/tests/test_hawkauth.py

```diff
@@ -13,6 +13,25 @@ def test_hawkauth_errors_when_credentials_and_hawk_session_passed(self):
     def test_hawkauth_errors_when_no_auth_is_set(self):
         self.assertRaises(AttributeError, HawkAuth)
 
+    def test_hawk_auth_using_credentials(self):
```

Comment:
It's a dict (at the moment) rather than parameters, would `..._as_dict` make more sense?